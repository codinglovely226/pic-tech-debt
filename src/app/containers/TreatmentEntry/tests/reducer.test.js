/* eslint-disable */

import { fromJS } from 'Immutable';
import TreatmentsReducer from '../reducer';

import {
  updateTreatments,
} from '../actions/case';


describe('TreatmentsReducer', () => {
  it('returns the initial state', () => {
    expect(TreatmentsReducer(undefined, {})).toMatchSnapshot();
  });
  it('update treatments', () => {
    var mockedDate = new Date(2018, 3, 7, 17, 15);
    global.Date = jest.fn(() => mockedDate);
    expect(TreatmentsReducer(fromJS({}), updateTreatments(1, 2, 'Test Disposition', {'key': 'Test Treatments'}, {'name': 'Test SPI Name'}))).toMatchSnapshot();
  });
});
