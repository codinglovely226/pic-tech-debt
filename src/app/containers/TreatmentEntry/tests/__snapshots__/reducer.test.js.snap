exports[`TreatmentsReducer returns the initial state 1`] = `Array []`;

exports[`TreatmentsReducer update treatments 1`] = `
Array [
  Object {
    "disposition": "Test Disposition",
    "notes": Array [
      2,
    ],
    "spi": Object {
      "id": undefined,
      "job": undefined,
      "name": "Test SPI Name",
      "org": undefined,
      "roles": undefined,
      "siteId": undefined,
    },
    "timestamp": "2018-04-07T17:15:00.000Z",
    "treatments": Object {
      "key": "Test Treatments",
    },
  },
]
`;
